src/vmdeps.txt

```diff
@@ -17,3 +17,5 @@ selinux-policy selinux-policy-targeted policycoreutils
 
 # coreos-assembler
 #FEDORA python3 python3-gobject-base buildah podman skopeo iptables iptables-libs
+
+gdisk xfsprogs e2fsprogs grub2 dosfstools grub2-efi-x64-modules
```

Comment:
Do you think this is necessary now or can it be in a follow up?

Reply:
Let's ensure the right people are pinged here and I think we can get some resources to do basic sanity checking.  Another good approach would be to just look at the partitions from a ppc64 install from Anaconda, etc.

Reply:
@ajeddeloh for ppc64le it should be as mentioned in https://github.com/coreos/fedora-coreos-tracker/issues/78#issuecomment-503571141

Reply:
Talked with some some folks about other arches and this will need conditionals in the future, but for now it's fine. Marking resolved for now and we'll revisit when switching more arches from anaconda.